lib/HTMLTree.php

```diff
@@ -98,7 +98,8 @@ public function draw($onlytree=false) {
 					$this->javascript .= '<div>';
 					$this->javascript .= '<input type="hidden" name="cmd" value="template_engine" />';
 					$this->javascript .= sprintf('<input type="hidden" name="server_id" value="%s" />',$server->getIndex());
-					$this->javascript .= sprintf('<input type="hidden" name="container" value="%s" />',htmlspecialchars($server->getContainer($base->getDN())));
+					$t = $server->getContainer($base->getDN());
+					$this->javascript .= sprintf('<input type="hidden" name="container" value="%s" />',htmlspecialchars(is_null($t)? '': $t));
```

Comment:
you can use `$this->javascript .= sprintf('<input type="hidden" name="container" value="%s" />',htmlspecialchars($server->getContainer($base->getDN() ?? '')));`

In what case did you encounter a null value ?

Reply:
I want to retain php 5.6 compatibility thus I don't use `??`

> In what case did you encounter a null value ?

I don't remember. exactly: I wrote it a long time ago and, as PR were not very responsive here, did not submit it.
AFAICR, I was experimenting with a DN component count <= 1.
See https://github.com/leenooks/phpLDAPadmin/blob/3c1f01622428e51e07029fcadf91f2519589d4b4/lib/ds_ldap.php#L1066-L1067